src/main/java/com/example/andersen/Task7/controller/UserController.java

```diff
@@ -0,0 +1,71 @@
+package com.example.andersen.Task7.controller;
+
+import com.example.andersen.Task7.model.User;
+import com.example.andersen.Task7.service.UserService;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/users")
+@AllArgsConstructor
+public class UserController {
+    private final UserService userService;
+
+    @GetMapping("/all")
+    public String getAll(Model theModel) {
+        List<User> users = userService.getAll();
+        theModel.addAttribute("users", users);
+        return "users_list";
+    }
+
+    @GetMapping("/create")
+    public String create(Model model) {
+        model.addAttribute("user", new User());
+        return "create-user";
+    }
+
+    @PostMapping("/create")
+    public String create(@Validated @ModelAttribute("user") User user, BindingResult result) {
+
+        if (result.hasErrors()) {
+            throw new IllegalArgumentException("User cannot be null");
+        }
+        User newUser = userService.create(user);
+        return "redirect:/orders/all/users/" + newUser.getId();
+    }
+
+    @GetMapping("/{id}/delete")
+    public String delete(@PathVariable("id") int id) {
+        userService.delete(id);
+        return "redirect:/users/all";
+    }
```

Comment:
Controller`s methods and mappings should be updated after switching on microservice architecture